docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 worker: &worker
-  image: addons/addons-server
+  image: addons/addons-server:feature_2176-django-1.8
```

Comment:
eh, yeah we don't - but it's unfortunately neccessary to get circleci builds working (which actually helped finding a few issues here) so I'd remove it right before merging